.github/workflows/test_python.yml

```diff
@@ -0,0 +1,29 @@
+name: test nodl_python
+
+on:
+  pull_request:
+    paths:
+      - '**.py'
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  build-and-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+      fail-fast: false
```

Comment:
To clarify, the docs say:

> When set to true, GitHub cancels all in-progress jobs if any matrix job fails. Default: true

This would e.g. cancel the Linux test if the Windows test failed, is that correct?

Reply:
Settings `fail-fast: false` disables that

Reply:
Okay very good, just wanted to make sure I understood properly.